client/src/related/relatedList.jsx

```diff
@@ -19,10 +20,16 @@ class RelatedList extends React.Component {
       ],
```

Comment:
I can't tell if GitHub is showing me something weird or you just have the tail end of a deleted array